atomicfu/src/commonMain/kotlin/kotlinx/atomicfu/OptionalJsName.kt

```diff
@@ -0,0 +1,14 @@
+package kotlinx.atomicfu
+
+@OptIn(ExperimentalMultiplatform::class)
+@OptionalExpectation
+@Retention(AnnotationRetention.BINARY)
+@Target(
+    AnnotationTarget.CLASS,
+    AnnotationTarget.FUNCTION,
+    AnnotationTarget.PROPERTY,
+    AnnotationTarget.CONSTRUCTOR,
+    AnnotationTarget.PROPERTY_GETTER,
+    AnnotationTarget.PROPERTY_SETTER
+)
+expect annotation class OptionalJsName(val name: String)
```

Comment:
It would be nice to have a comment clarifying purpose of this annotation (i.e. it's aliased by the `JsName` when compiled to JS and it's NOOP when compiled to Wasm).
Otherwise it's unclear why the name is optional.

Reply:
Done